src/pykeen/datasets/ckg.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+"""Clinical Knowledge Graph."""
+
+import tarfile
+from pathlib import Path
+from typing import Iterable
+
+import click
+import pandas as pd
+import pystow
+
+from pykeen.datasets.base import TabbedDataset
+
+__all__ = [
+    'CKG',
+]
+
+URL = 'https://md-datasets-public-files-prod.s3.eu-west-1.amazonaws.com/d1e8d3df-2342-468a-91a9-97a981a479ad'
+
+
+class CKG(TabbedDataset):
+    """The Clinical Knowledge Graph (CKG) dataset from [santos2020]_.
+
+    This dataset contains ~7.6 million nodes, 11 relations, and ~26 million triples.
+
+    .. [santos2020] Santos, A., *et al* (2020). `Clinical Knowledge Graph Integrates Proteomics Data into Clinical
+       Decision-Making <https://doi.org/10.1101/2020.05.09.084897>`_. *bioRxiv*, 2020.05.09.084897.
+    """
+
+    def _get_df(self) -> pd.DataFrame:
+        return _get_df()
```

Comment:
why is this a separate method?

Reply:
good point. was testing before and didn't combine it

Reply:
Fixed in f78d4c7b